tests/jobs/test_triggerer_job.py

```diff
@@ -448,7 +452,7 @@ def test_invalid_trigger(session, dag_maker):
     job.load_triggers()
 
     # Make sure it turned up in the failed queue
-    assert list(job.runner.failed_triggers) == [1]
+    assert len(job.runner.failed_triggers) == 1
```

Comment:
isn't the before test stricter?

Reply:
Yeah, the before test was stricter. But now failed triggers are a 2-tuple so if we test the first component, do we then need to test the other one as well etc – to my mind, these tests are just heuristics trying to help us prevent bugs, but they can't be exhaustive.

But I can make it as strict as before.

Reply:
```python
assert len(job.runner.failed_triggers) == 1
assert job.runner.failed_triggers[0][0] == 1
```

Is good enough IMO.

Reply:
Fixed in c891e106905a864c3aa5da57cd8495449e4d3b65.